homeassistant/components/webostv/strings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "config": {
+    "flow_title": "LG webOS Smart TV",
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "description": "Turn on WebosTV device, fill fields and submit",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "name": "[%key:common::config_flow::data::name%]",
+          "icon": "Icon - optional (ex: mdi:tv)"
```

Comment:
The config flow data should only collect enough to connect/setup the device.  Anything that isn't needed to connect belongs in the options flow.

We shouldn't allow users to configure the icon. We prefer setting a device class when possible